apps/ensindexer/.env.local.example

```
@@ -166,27 +166,26 @@
+# https://ponder.sh/docs/api-reference/ponder/database#database-schema-rules
 #
 # Avoid using the `public` schema as we force that in the `dev` command. Using `public`
 # cause conflicts as you interchange between dev and start commands so use literally
 # anything else.
-DATABASE_SCHEMA=production
-# Required. This is the connection string for the database that the indexer will use to store data.
+ENSINDEXER_SCHEMA_NAME=ensindexer_0
+# Required. This is the connection string for the Postgres database where ENSIndexer will store data.
 # It should be in the format of `postgresql://<username>:<password>@<host>:<port>/<database>`
-DATABASE_URL=postgresql://dbuser:abcd1234@localhost:5432/my_database
+ENSDB_URL=postgresql://dbuser:abcd1234@localhost:5432/my_database
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENSDB_URL key should go before the ENSINDEXER_SCHEMA_NAME key

(UnorderedKey)